crates/ide_assists/src/handlers/extract_function.rs

```diff
@@ -10,15 +10,8 @@ use ide_db::{
 use itertools::Itertools;
 use stdx::format_to;
 use syntax::{
-    ast::{
-        self,
-        edit::{AstNodeEdit, IndentLevel},
-        AstNode,
-    },
-    ted,
-    SyntaxKind::{self, BLOCK_EXPR, BREAK_EXPR, COMMENT, PATH_EXPR, RETURN_EXPR},
-    SyntaxNode, SyntaxToken, TextRange, TextSize, TokenAtOffset, WalkEvent, T,
-};
+    ast::{self, AstNode, edit::{AstNodeEdit, IndentLevel}},
+    SyntaxKind::{self, BLOCK_EXPR, BREAK_EXPR, COMMENT, PATH_EXPR, RETURN_EXPR}, SyntaxNode, SyntaxToken, T, TextRange, TextSize, TokenAtOffset, WalkEvent, ted};
```

Comment:
Aren't we checking formatting on CI? How is this not failing here? 😅

Reply:
`cargo fmt` seems to be fine with this formatting.

Reply:
Oh is this one of those cases were rustfmt just accepts the files and ignores processing it? .... Not the first time I encountered this...

Reply:
Actually, it might be a macro call in the file that's longer than the max char per line setting where it doesn't know how to format it, I think thats usually what causes it to just bail out

Reply:
Yeah, apparently `cargo +stable fmt` completely ignores all files in `ide_assists/src/handlers` and `ide_diagnostics/src/handlers`. Maybe it's something about the way these modules are declared? It seems to be fixed in current nightly though, so I guess we'll just have to wait until the fix becomes stable...